nx/test/nx/serving_test.exs

```diff
@@ -115,6 +115,34 @@ defmodule Nx.ServingTest do
       assert batch.pad == 0
       assert Nx.Defn.jit_apply(&Function.identity/1, [batch]) == Nx.tensor([[1, 2], [3, 4]])
     end
+
+    @tag :capture_log
```

Comment:
`attach_event_handlers` triggers telemetry's "anonymous function" warning, hence capturing the logs. The warning should be silenced/avoided by the test helper. I think the warning would be confusing if you haven't worked with telemetry handlers before.

Reply:
1.2.1 is out with a fix for that :)

Reply:
Indeed. Updating it 👍